hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -3356,4 +3356,15 @@ List<LogEntry> getLogEntries(Set<ServerName> serverNames, String logType, Server
    * Get the list of cached files
    */
   List<String> getCachedFilesList(ServerName serverName) throws IOException;
+
+  @InterfaceAudience.Private
+  default void restoreBackupSystemTable(String snapshotName) throws IOException {
+    SnapshotDescription snapshot =
+      listSnapshots().stream().filter(s -> s.getName().equals(snapshotName)).findFirst()
+        .orElseThrow(() -> new IOException("Snapshot " + snapshotName + " not found"));
+    TableName tn = snapshot.getTableName();
+    disableTable(tn);
```

Comment:
This is the same behavior as the logic we had in the BackupSystemTable that we're replacing. Does it makse sense to include this behavior? Or should this method do nothing by default?